src/Image.php

```diff
@@ -25,6 +25,15 @@ class Image extends File
      */
     private static $plural_name = "Images";
 
+    /**
+     * Globally control whether Images added via the WYSIWYG editor or inserted as Image objects in Silverstripe
+     * templates have the loading="lazy" HTML added by default
```

Comment:
Did we decide that the default value is not configurable?

If not, then the above statement is incorrect.
Otherwise, where did we decide on that?

Reply:
Yes, has been discussed a few times and decision has been made to make the default non-configurable.  See https://github.com/silverstripe/silverstripe-assets/issues/453